src/mumble/SettingsKeys.h

```diff
@@ -169,7 +169,7 @@ const SettingsKey CHANNEL_EXPANSION_MODE_KEY           = { "channel_expansion_mo
 const SettingsKey CHANNEL_DRAG_MODE_KEY                = { "channel_drag_mode" };
 const SettingsKey USER_DRAG_MODE_KEY                   = { "user_drag_mode" };
 const SettingsKey ALWAYS_ON_TOP_KEY                    = { "always_on_top" };
-const SettingsKey ASK_ON_QUIT_KEY                      = { "ask_on_quit" };
+const SettingsKey CONNECTED_QUIT_BEHAVIOUR_KEY         = { "connected_quit_behaviour"};
```

Comment:
Normally we stick to `behavior` rather than `behaviour` (though both are technically correct - it's just a Brits vs 'mericans thing)
```suggestion
const SettingsKey CONNECTED_QUIT_BEHAVIOR_KEY         = { "connected_quit_behavior"};
```